lib/ace/ace.js

```diff
@@ -55,6 +55,7 @@ require("./multi_select");
 require("./mode/folding/fold_mode");
 require("./theme/textmate");
 require("./ext/error_marker");
+require("./lib/katex/katex")
```

Comment:
@dbarashev but `require` is not included in this page, and I thought, adding KaTeX to the build meant making it so that it is actually a part of ACE.